client/src/components/ResultIntervalPlot/index.jsx

```diff
@@ -4,7 +4,11 @@ import Highcharts from 'highcharts';
 import highchartsMore from 'highcharts/highcharts-more';
 import HighchartsReact from 'highcharts-react-official';
 
-const ResultIntervalPlot = ({ schedule }) => {
+import algorithms from '../../config/algorithms.json';
+
+const algorithmKeys = Object.values(algorithms).map(({ key }) => key);
```

Comment:
why do we need Object.values here?

Reply:
We are using `key` value from the inner object as an ID for algorithm